### ISSUE TITLE
Build FAIL on Arch Linux

### ISSUE THREAD

**Issue by giappi** [calleerlandsson/pick#281]
```
==================================
   pick 2.0.1: ./test-suite.log
==================================

# TOTAL: 33
# PASS:  32
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0

.. contents:: :depth: 2

FAIL: tests/key-printable
=========================

FAIL:	printable UTF-8 characters are added to the query
CAUSE:	wrong output
--- stdout-want
+++ stdout-got
@@ -1 +1 @@
-��
+��
FAIL:	UTF-8 false positive
CAUSE:	wrong output
--- stdout-want
+++ stdout-got
@@ -1 +1 @@
-h��h h��h
+���� ��
FAIL:	UTF-8 four byte wide characters
CAUSE:	wrong output
--- stdout-want
+++ stdout-got
@@ -1 +1 @@
-����
+����
FAIL tests/key-printable.t (exit status: 1)

```

**Comment by mptre** [comment]
This is tricky to debug remotely. But, what's the output of the
following commands:

```sh
$ cat <<EOF >test.c
#include <stdlib.h>
#include <stdio.h>

int
main(void)
{
	printf("MB_CUR_MAX=%ld\n", MB_CUR_MAX);
	return 0;
}
EOF
$ cc -o test test.c && ./test; locale
```

**Comment by DBOTW** [comment]
Seems like an `locale`-issue: can reproduce this with `LC_ALL=C make check` on ArchLabs ...

**Comment by mptre** [comment]
@DBOTW maybe we should set the remaining `LC_*` variables in
[pick-test]. Feel free to test and submit a PR :-)

[pick-test]: https://github.com/calleerlandsson/pick/blob/master/tests/pick-test.c#L29

**Comment by DBOTW** [comment]
@mptre just setting `LC_CTYPE` (or other `LC_`-variables) to `C` does not break `make check`, only the `LC_ALL`-variable does. The Debian-Maintainer sets `LC_ALL` explicit to `C.UTF-8` (Debian-specific?): https://github.com/eavgerinos/pkg-pick/commit/d3787b74569f3ccb67553c7ced2a6ed34f62e3d0#diff-d2b2229a977ef7e99780f2207c29b503

**Comment by mptre** [comment]
Should be fixed by 43a9f02. I'm planning on doing a patch release
hopefully later this week.
